pkg/webhook/webhook.go

```diff
@@ -114,6 +119,7 @@ func (wh *WebHook) Stop(webhookConfigName string) error {
 }
 
 func (wh *WebHook) serve(w http.ResponseWriter, r *http.Request) {
+	glog.Info("Serving admission request")
```

Comment:
Can we change this to `glog.V(2).Info()`?

Reply:
Done.